src/mongo/db/storage/kv/dictionary/kv_sorted_data_impl.cpp

```diff
@@ -89,7 +89,7 @@ namespace mongo {
                                        const IndexDescriptor* desc)
         : _db(db),
           _ordering(Ordering::make(desc ? desc->keyPattern() : BSONObj())),
-          _dupsAllowed(false)
+          _dupsAllowed(true)
```

Comment:
@denis-protyvenskyi @christianrober  yes, I like this idea, will prepare alternative pull request for review.